src/main/java/frc/robot/commands/intake/IntakeAndIndex.java

```diff
@@ -0,0 +1,39 @@
+package frc.robot.commands.intake;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.ScheduleCommand;
+import frc.robot.subsystems.IndexerSubsystem;
+import frc.robot.subsystems.IntakeSubsystem;
+import friarLib2.commands.RunUntilCommand;
+
+/**
+ * While this command is running, run the intake. When this command 
+ * starts, tell the indexer to index a cargo.
+ */
+public class IntakeAndIndex extends ParallelCommandGroup {
+
+    private Timer timer = new Timer();
+
+    public IntakeAndIndex (IntakeSubsystem intake, IndexerSubsystem indexer) {
+        timer.reset();
+        timer.stop();
```

Comment:
Are we starting the timer somewhere?

Reply:
Yes, in the `end()` method at the bottom of this file (when the `IntakeAndIndex` command group ends).